betterave-backend/betterave_backend/app/operations/notification_operations.py

```diff
@@ -0,0 +1,139 @@
+# type: ignore
+from typing import Optional
+from flask import request
+from sqlalchemy.exc import SQLAlchemyError
+from betterave_backend.extensions import db
+from betterave_backend.app.models import Notification, User, UserLevel
+from betterave_backend.app.decorators import is_valid_apikey, with_instance
+
+
+def add_notification(
+    title: str,
+    content: str,
+    sent_by_user_id: int,
+    recipient_type,
+) -> int:
+    """Add a notification to the database."""
+    try:
+        if recipient_type == "Subscribers":
+            recipient_users = User.query.get(sent_by_user_id).subscribers
+        elif recipient_type == "All users":
+            recipient_users = User.query.all()
+        else:
```

Comment:
Maybe use `elif isinstance(recipient_type, UserLevel)` and throw error on `else`